hooks/command

```diff
@@ -182,6 +182,9 @@ status=""
 if [[ "$jobstatus" == "Complete" ]] ; then
   echo "success"
   status="0"
+elif kubectl get pod "$pod_name" 2>&1 >/dev/null | grep -q NotFound; then
+  echo "Warning: pod was never created"
```

Comment:
```suggestion
  echo "Warning: pod not found"
```
to eliminate the rare case where the pod for whatever reason was deleted before we had a chance to check it